From 2578b71bbf70d7459fde5971956420bf48ae815b Mon Sep 17 00:00:00 2001
From: Hong H. Pham <hong.pham@windriver.com>
Date: Tue, 20 Jan 2009 10:15:20 -0500
Subject: [PATCH] preempt_irq: remove NO__DO_IRQ from sparc.

Remove GENERIC_HARDIRQS_NO__DO_IRQ, which is an anti-requisite for
PREEMPT_HARDIRQS, in the SPARC64 specific Kconfig.

Signed-off-by: Hong H. Pham <hong.pham@windriver.com>
---
 arch/sparc64/Kconfig |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/arch/sparc64/Kconfig b/arch/sparc64/Kconfig
index beae7fe..18f48cc 100644
--- a/arch/sparc64/Kconfig
+++ b/arch/sparc64/Kconfig
@@ -88,10 +88,6 @@ config ARCH_NO_VIRT_TO_BUS
 config OF
 	def_bool y
 
-config GENERIC_HARDIRQS_NO__DO_IRQ
-	bool
-	def_bool y
-
 source "init/Kconfig"
 
 menu "Processor type and features"
-- 
1.6.0.3

